### DISCUSSION TITLE
Summit photos

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#149] [General]
A thread for photos from the summit! Mine are landing here:

https://erbonzo.smugmug.com/Travel/DELPH-IN-2024

**Comment by fcbond** [comment]
Thanks.

On Thu, 4 Jul 2024 at 09:56, emilymbender ***@***.***> wrote:

> A thread for photos from the summit! Mine are landing here:
>
> https://erbonzo.smugmug.com/Travel/DELPH-IN-2024
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/149>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRQBDN7ECVHJS4NUI6TZKT5ZRAVCNFSM6AAAAABKK74RUCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHA4TMMJZGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by sanghoun** [comment]
Dear folks,

My photos are downloadable from here.
https://drive.google.com/drive/folders/1n_wYNV57rxsBUHahGKX1RPj_8VcHAGFi?usp=drive_link

I was so happy during the DELPH-IN this year. Thanks again to everybody!

Sanghoun
=============================
Sanghoun Song
Associate Professor
Dept. of Linguistics, Korea University
http://corpus.mireene.com
phone: +82-2-3290-2177 (office)
=============================


2024년 7월 4일 (목) 오후 6:14, Francis Bond ***@***.***>님이 작성:

> Thanks.
>

**Comment by emilymbender** [comment]
Thank you, Sanghoun! It was good to see you :)

**Comment by olzama** [comment]
My photos are here: https://photos.app.goo.gl/8mDjqHTkrL9JAxgU9

**Comment by emilymbender** [comment]
Thanks, Olga!
